### ISSUE TITLE
Allow verification of client

### ISSUE THREAD

**Issue by HoffmannP** [matrix-org/purple-matrix#123]
In element one can verify devices, I'm not sure if this is linked to the encryption funktionality or even follows any standard. But it would be nice if there'd be a way to accept the verification request.

**Comment by richvdh** [comment]
yes it is part of the encryption functionality, and hence part of #18.
